### PR TITLE
Merged latestTime, increaseTime and duration into a single time helper.

### PR DESCRIPTION
**Pull request by nventuro**
Could be considered part of #1362.

### CODE REVIEW

File: test/token/ERC20/TokenVesting.test.js [hunk 1]
```diff
@@ -18,9 +17,10 @@ contract('TokenVesting', function ([_, owner, beneficiary]) {
   const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
 
   beforeEach(async function () {
-    this.start = (await latestTime()) + duration.minutes(1); // +1 minute so it starts after contract instantiation
-    this.cliffDuration = duration.years(1);
-    this.duration = duration.years(2);
+    // +1 minute so it starts after contract instantiation
```
Comment (frangio) [inline test/token/ERC20/TokenVesting.test.js]: This sort of thing feels brittle. I'm not saying we change it here, but now that our eyes are on it, can we think of alternative ways to achieve the same thing?

Comment (frangio) [reply test/token/ERC20/TokenVesting.test.js]: Also thinking about how we can make the whole time testing work on an actual node at some point... Perhaps the more robust thing would be to have a mock of a timestamp `ts` for which `ts.greaterThan(now)` is always true.

Comment (nventuro) [reply test/token/ERC20/TokenVesting.test.js]: Woulnd't the mock thing need to happen at the node, though? I'm not sure how we can get around the contract executing the `TIMESTAMP` instruction from the suite.

Comment (frangio) [reply test/token/ERC20/TokenVesting.test.js]: One idea that's been going around is to have a `Time` base contract that all contracts inherit which can be swapped out for a mock... See #355.

**Pull request merged**